ocaml/xapi/valid_ref_list.ml

```diff
@@ -16,3 +16,5 @@ let map f = Stdext.Listext.List.filter_map (default_on_missing_ref (fun x -> Som
 
 let flat_map f l = List.map (default_on_missing_ref f []) l |> List.flatten
 
+let filter_map f l = map f l |> filter ((<>) None) |> map Xapi_stdext_monadic.Opt.unbox
```

Comment:
For the `filter` and the second `map` we can use the versions from `List`.

Reply:
Good idea.